packages/expressive-code/README.md

```diff
@@ -50,13 +54,38 @@ const { renderedGroupAst, styles } = await ec.render({
 let htmlContent = toHtml(renderedGroupAst)
 
 // Collect styles and add them before the HTML content
-const stylesToPrepend: string[] = []
+const stylesToPrepend = []
 stylesToPrepend.push(baseStyles)
-stylesToPrepend.push(...styles)
-if (stylesToPrepend.length) {
-  htmlContent = `<style>${[...stylesToPrepend].join('')}</style>${htmlContent}`
-}
-
+stylesToPrepend.push(themeStyles)
+stylesToPrepend.push(...blockStyles)
+
+const styleContent = `<style> ${[...stylesToPrepend].join('')} </style>`
+const jsContent = `<script type="module"> ${[...jsModules].join('')} </script>`
+
+const htmlDocument = `
+<!doctype html>
+<html lang="en" data-theme="light">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
```

Comment:
```suggestion
    <meta name="viewport" content="width=device-width, initial-scale=1">
```

As this project cares a lot about accessibility, I'd not like to include an example that restricts user scaling of the page. Specifying the initial scale should be fine, but I'd prefer to remove the other settings.